Dockerfile

```diff
@@ -0,0 +1,21 @@
+#
+# BUILD CONTAINER
+#
+FROM node:16 as base
+USER node
+ENV NODE_ENV production
+WORKDIR /usr/src/app
+COPY --chown=node:node package.json yarn.lock .yarnrc.yml tsconfig*.json ./
```

Comment:
ooc: does copying `.yarnrc.yml` change the outcome of the image? I was under the impression that `corepack` had to be enabled for this to work and I'm not sure this is the case with this image.

Reply:
After inspecting the image it does seem that it doesn't run with the bundled yarn version 🤔 

```
$ yarn --version
1.22.19
```

Reply:
Actually with the following two lines I'd expect it to use the bundled one

```
COPY --chown=node:node .yarn/releases ./.yarn/releases
RUN yarn install --immutable 
```

Can you confirm this behaviour?

Reply:
Good catch, I didn't verify this was using Yarn 3.X, I will research it and update the PR, thanks.

Reply:
It's using Yarn 3.2.2. I added a step to the build container to check it:
```
RUN yarn --version
```
And this is the output.
```
#11 [base 5/8] RUN yarn --version
#11 sha256:6f7a19c95f998d36997d3f1ce06e6242edeb00b0837490b4b8cd06c13d87f41d
#11 0.438 3.2.2
#11 DONE 0.5s
``` 
The Yarn version it's different in the prod container as we suspected (and of course it doesn't matter 🙂).